Dockerfile

```diff
@@ -0,0 +1,21 @@
+# IMAGE FOR BUILDING
+FROM golang:1.19.2 as builder 
+
+WORKDIR /app
+
+COPY . /app
```

Comment:
Let's rename this to `dev.Dockerfile`? Given it can mainly be used for building what you have _locally_ into a docker file? For example, it doesn't support doing something like checking out a particular release, which means it's less useful for doing things like automated release builds. 

If you take a look at the main `Dockerfile` in the `lnd` repo, it has some of these features that even let you build from a fork as well: https://github.com/lightningnetwork/lnd/blob/master/Dockerfile#L9-L27

Reply:
alright. for now let's rename it to dev.Dockerfile. Maybe I will start a new PR with a main Dockerfile after this one is done and no one else has done that already then.